tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>"map this over" -&gt; "map this function over"</p><p> </p><p>I wonder of the details of 1 and 2 would be better presented in the docstring of the function. Writing function <a href="https://google.github.io/styleguide/pyguide.html#381-docstrings" target="_blank">docstrings</a> is a best practice I think we should encourage.</p><p> </p><p>Suggest explaining why <code>.set({'millis': img.date().millis()</code>is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done.</p>